frontend/src/pages/meditation/components/meditation-player/meditation-player.tsx

```diff
@@ -1,41 +1,50 @@
 import { AudioPlayer } from '#libs/components/components.js';
-import { useCallback, useState } from '#libs/hooks/hooks.js';
+import { useAppSelector, useCallback, useState } from '#libs/hooks/hooks.js';
 import {
-  mockedPlaylist,
+  MOCKED_IMAGE,
   TRACK_FIRST_INDEX,
-} from '#pages/meditation/constants/constants.js';
+} from '#pages/meditation/libs/constants/constants.js';
 
 import styles from './styles.module.scss';
 
 const MeditationPlayer: React.FC = () => {
+  const { meditationEntries } = useAppSelector(({ meditation }) => {
+    return {
+      meditationEntries: meditation.meditationEntries,
+    };
+  });
   const [trackIndex, setTrackIndex] = useState(TRACK_FIRST_INDEX);
-  const [currentTrack, setCurrentTrack] = useState(mockedPlaylist[trackIndex]);
+  const [currentTrack, setCurrentTrack] = useState(
+    meditationEntries[trackIndex],
+  );
 
-  const handleTrackIndex = useCallback((index: number): void => {
-    setTrackIndex(index);
-    setCurrentTrack(mockedPlaylist[index]);
-  }, []);
+  const handleTrackIndex = useCallback(
+    (index: number): void => {
+      setTrackIndex(index);
+      setCurrentTrack(meditationEntries[index]);
+    },
+    [meditationEntries],
+  );
 
   return (
     <div className={styles['wrapper']}>
       <div className={styles['meditation-player']}>
         <div className={styles['image-wrapper']}>
           <img
             className={styles['image']}
-            src={currentTrack?.img}
+            src={MOCKED_IMAGE}
```

Comment:
import it in this file
```suggestion
            src={deafultMeditationImage}
```

Reply:
Done